main.go

```diff
@@ -221,21 +222,28 @@ func before(c *cli.Context) (*state.HelmState, helmexec.Interface, error) {
 	quiet := c.GlobalBool("quiet")
 	kubeContext := c.GlobalString("kube-context")
 
-	state, err := state.ReadFromFile(file)
+	st, err := state.ReadFromFile(file)
+	if err != nil && strings.Contains(err.Error(), fmt.Sprintf("open %s:", DefaultHelmfile)) {
+		var fallbackErr error
+		st, fallbackErr = state.ReadFromFile(DeprecatedHelmfile)
```

Comment:
when falling back to the deprecated value should we log a message to tell them it's deprecated?

Reply:
Good point! I believe so.

Reply:
Added to https://github.com/roboll/helmfile/pull/35/files#diff-7ddfb3e035b42cd70649cc33393fe32cR249